user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_ad_hoc_data.md

```diff
@@ -14,7 +14,13 @@ An ad hoc data source retrieves external data based on an Automation script that
 
 For more information on how you can create ad hoc data sources, see [Ad hoc data sources](xref:GQI_Ad_hoc_data_sources).
 
-This query data source is available from DataMiner 10.3.0/10.2.4 onwards.
+This query data source is available from DataMiner 10.2.4/10.3.0 onwards.
+
+## Linking the data source to data in the same dashboard or low-code app
```

Comment:
It's  bit weird to call it "data in the same dashboard or app", we typically call that "data of the dashboard or app"